src/Connector.sol

```diff
@@ -134,14 +134,14 @@ contract CentrifugeConnector {
         memberlist.updateMember(user, validUntil);
     }
 
-    function handleTransfer(uint64 poolId, bytes16 trancheId, address user, uint256 amount) public onlyRouter {
+    function handleTransfer(uint64 poolId, bytes16 trancheId, address user, uint128 amount) public onlyRouter {
```

Comment:
@mustermeiszer @offerijns I also realised that this API won't allow to transfer tokens to an actual centrifuge address, since that currently the API expects an Ethereum `address` that we suffix with 12 zeros for Centrifuge compatibility.

Reply:
`handleTransfer` is only for incoming transfers though, right? Which always should go to an Ethereum `address`

Reply:
`transfer` definitely should accept a `bytes32` based address though

Reply:
Yeah sorry, left the comment in the wrong place 👍🏻